great_expectations/core/usage_statistics/anonymizers/profiler_run_anonymizer.py

```diff
@@ -0,0 +1,196 @@
+
+        batch_request: Optional[dict] = domain_builder.get("batch_request")
+        if batch_request:
+            anonymized_domain_builder[
+                "anonymized_batch_request"
+            ] = self._batch_request_anonymizer.anonymize_batch_request(
+                *(), **batch_request
+            )
+            logger.debug("Anonymized batch request in DomainBuilder")
+
+        return anonymized_domain_builder
+
+    def _anonymize_parameter_builders(
+        self, parameter_builders: List[dict]
+    ) -> List[dict]:
+        anonymized_parameter_builders: List[dict] = []
+
+        for parameter_builder in parameter_builders:
+            anonymized_parameter_builder: dict = self._anonymize_parameter_builder(
+                parameter_builder
+            )
+            anonymized_parameter_builders.append(anonymized_parameter_builder)
+
+        return anonymized_parameter_builders
+
+    def _anonymize_parameter_builder(self, parameter_builder: dict) -> dict:
+        anonymized_parameter_builder: dict = {}
+
+        anonymized_parameter_builder["anonymized_name"] = self.anonymize(
+            parameter_builder.get("name")
+        )
+
+        anonymized_parameter_builder["class_name"] = parameter_builder.get("class_name")
+
+        batch_request: Optional[dict] = parameter_builder.get("batch_request")
+        if batch_request:
+            anonymized_parameter_builder[
+                "anonymized_batch_request"
+            ] = self._batch_request_anonymizer.anonymize_batch_request(
+                *(), **batch_request
+            )
+            logger.debug("Anonymized batch request in ParameterBuilder")
+
+        return anonymized_parameter_builder
+
+    def _anonymize_expectation_configuration_builders(
+        self, expectation_configuration_builders: List[dict]
+    ) -> List[dict]:
+        anonymized_expectation_configuration_builders: List[dict] = []
+
+        for expectation_configuration_builder in expectation_configuration_builders:
+            anonymized_expectation_configuration_builder: dict = (
+                self._anonymize_expectation_configuration_builder(
+                    expectation_configuration_builder
+                )
+            )
+            anonymized_expectation_configuration_builders.append(
+                anonymized_expectation_configuration_builder
+            )
+
+        return anonymized_expectation_configuration_builders
+
+    def _anonymize_expectation_configuration_builder(
+        self, expectation_configuration_builder: dict
+    ) -> dict:
+        anonymized_expectation_configuration_builder: dict = {}
+
+        anonymized_expectation_configuration_builder[
+            "class_name"
+        ] = expectation_configuration_builder.get("class_name")
+        anonymized_expectation_configuration_builder[
+            "expectation_type"
+        ] = expectation_configuration_builder.get("expectation_type")
```

Comment:
This expectation_type will also need to be anonymized if it is not one of the GE provided expectation types. See note above.